Cargo.toml

```diff
@@ -8,8 +8,9 @@ edition = "2018"
 clap = { version = "2.33.0", features = ["yaml"] }
 zip = "0.5.3"
 ignore = "0.4"
-globset = "0.3"
-
+# globset = "0.3"
```

Comment:
commented it out until we enable this and uncomment the code in `options` module